runtime/mux_test.go

```diff
@@ -719,6 +719,34 @@ func TestWithHealthzEndpoint_serviceParam(t *testing.T) {
 	}
 }
 
+func TestWithHealthzEndpoint_header(t *testing.T) {
+	for _, tt := range healthCheckTests {
+		t.Run(tt.name, func(t *testing.T) {
+			mux := runtime.NewServeMux(runtime.WithHealthzEndpoint(&dummyHealthCheckClient{status: tt.status, code: tt.code}))
+
+			r := httptest.NewRequest(http.MethodGet, "/healthz", nil)
+			rr := httptest.NewRecorder()
+
+			mux.ServeHTTP(rr, r)
+
+			// Expected header values in the HTTP response
+			expectedHeaderValue := map[string]string{
+				"Content-Type": "application/json",
+				//"Grpc-Metadata-Content-Type": "application/grpc",
```

Comment:
Why is this commented out?

Reply:
This was just an example to show that you can easily add more or less headers to test.
And `Grpc-Metadata-Content-Type` can't pass because it is deleted by the errorHandler (so the test wont pass).
If I uncomment the `Grpc-Metadata` line all tests pass except the 404 one

Reply:
While there is only one header left to test. Would you like that i change the map+loop to something lighter?

Reply:
Sure, lets just do a hardcoded check, thanks.